bench/workgen/workgen_int.h

```diff
@@ -246,9 +246,11 @@ struct OperationInternal {
 };
 
 struct CheckpointOperationInternal : OperationInternal {
-    CheckpointOperationInternal() : OperationInternal() {}
+    char const * ckpt_config;
```

Comment:
> Could we make this typed std::string ? Otherwise I would worry about the storage for it to be deallocated after `CheckpointOperationInternal::parse_config` is called and before `heckpointOperationInternal::run` is called.

Yeah, I agree with the char* we need to worry about the deallocation. Updated the member variable ckpt_config to std::string.